tutorials/btp-integration-suite-nonsapconnectivity-usecase/btp-integration-suite-nonsapconnectivity-usecase.md

```diff
@@ -8,35 +8,36 @@ author_name: Karunaharan V
 author_profile: https://github.com/Karunaharan
 ---
 
-# The Use Case for the Mission
-<!-- description --> Understand the integration problem that you're going to solve before you get started
+# The Use Case: Connectivity to Non-SAP Applications using SAP Integration Suite
+<!-- description --> Understand the integration problem that you're going to solve before you get started.
+
 
 ## You will learn
  - What is SAP Integration Suite
- - The integration problem that you're going to solve
+ - About the integration problem that you're going to solve
 
----
 
 ### What is SAP Integration Suite
```

Comment:
As there are no actions required by the user here, I recommend making this part of the Intro section.

## Intro
(content goes here)